jetty-server/src/main/config/modules/bytebufferpool.mod

```diff
@@ -1,5 +1,6 @@
 [description]
 Configures the ByteBufferPool used by ServerConnectors.
+Use module "bytebufferpool-logarithmic" for a more space efficient pool.
```

Comment:
Well more space efficient in the pool, but may be less space efficient in the buffer itself.

For the sizes we are commonly using, I'm actually thinking that just increasing the default factor to 4K is pretty good..... except for tiny buffers.  Geting 1K when you only needed 6 bytes was bad enough, but 4K get's a bit ridiculous.

Reply:
"less space efficient in the buffer" is indeed what my comment is about and your remark about tiny buffers is exactly my point, but it also applies to other scenarios.

When you consider that in a typical SSL scenario where the buffer pool is limited in size, configuring a logarithmic pool may very well be noticeably less efficient; i.e.: a pool configured to hold a maximum of 16 MB is only going to be able to hold at most ~500 SSL buffers when using a logarithmic divide while it could hold up to ~950 with a linear divide.

I'm just not sure if it's a real issue in practice, so I'm conservatively thinking that we should not change the default nor recommend alternatives until we have a clearer view.